be/src/service/doris_main.cpp

```diff
@@ -488,6 +465,28 @@ int main(int argc, char** argv) {
     }
 #endif
 
+    // open pid file, obtain file lock and save pid
```

Comment:
as we need to move the code location, is it better wrap these codes into a function?